woocommerce/wc-admin
# Migrate navigation store to data package

Fixes #5351 

Migrates nav store to data package.

### Detailed test instructions:

1. Enable the nav.
1. Navigate to various pages using the nav.
1. Make sure route detection works as expected and active item is correctly marked active.
1. Make sure tests pass.
